_config.yml

```
@@ -2,7 +2,7 @@
 # ---------------------------------------------------------------
 name: Typelevel
 title: Typelevel
-description: Let the Scala compiler work for you. We provide type classes, instances, conversions, testing, supplements to the standard library, and much more.
+description: Typelevel is an association of projects and individuals united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.

```

idk. "association" sounds so formal, like The Typelevel Association. Are we an association?

```suggestion
description: Typelevel is an group of projects and individuals united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.

```

---

Combining a bit with your other comment, what about:

> Typelevel is a community of people and projects united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.

---

Actually "community of people" is kind of redundant, and you can't really have a community of projects...
:thinking:

---

> Typelevel is an ecosystem of projects and community united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.

???

---

idk what's best, but perhaps "Typelevel is a collective dedicated to..."